package-lock.json

```diff
@@ -1,1521 +1,39 @@
 {
   "name": "teamwilder",
   "version": "0.0.0",
-  "lockfileVersion": 2,
+  "lockfileVersion": 1,
```

Comment:
SHould be removed from this PR. I think the addition was related to the duplicate dependency of elm-format.